### ISSUE TITLE
[suggestion] Антагонисты невосприимчивы к способностям вампиров

### ISSUE THREAD

**Issue by Filatelele** [ChaoticOnyx/OnyxBay#10831]
### Описание бага
ВСЕ антагонисты невосприимчивы к большинству способностей вампиров. Даже трейторы. Даже революционеры. 
Прожимаешь glare+screech и думаешь, что сейчас завалишь ассистуху? Серый трейтор думает иначе. 

Абилки вампира вызывают проверку can_affect(), которая возвращает false если цель обладает любой ролькой. 
vampire_datum.dm 
234 строчка

`if(is_special_character(target))
		// The target is some non-vampire antag
		NOTIFIED_WARNING("[target]'s mind is too strong to be affected by our powers!")
		return FALSE`

### Воспроизведение
Пытаемся использовать абилки на любом антаге. 

### Ожидаемое поведение
Можно применить способности вампира на цель. Возможно, некоторые роли действительно должны быть невосприимчивы - условные генокрады скажем. 
На мой взгляд, нет ни геймдизайновых, ни лоровых причин давать тем же трейторам и революционерам защиту от способностей вампира. 
Ну и способности типа screech наверное должны трогать всех, вне зависимости от роли.

**Comment by TobyThorne** [comment]
опять улучшения под видом багов протаскивают

#10795 не менял эту проверку, она и была в таком виде

**Comment by Filatelele** [comment]
> опять улучшения под видом багов протаскивают
> 
> #10795 не менял эту проверку, она и была в таком виде

Да, виноват. Надо меньше верить людям в дискорде и самому больше читать код. Стыдно. 


Переделал ишшуй в suggestion, ибо проблема все еще существует.
